src/more-infos/more-info-configurator.html

```diff
@@ -129,11 +149,38 @@
     return stateObj.state === 'configure';
   }
 
+  setDefaultFieldInputs(stateObj) {
+    var values = {};
+    for (var field of this.stateObj.attributes.fields) {
+      if (typeof field.value === 'undefined') {
+        continue;
+      }
+      values[field.id] = field.value;
+    }
+    return values;
+  }
+
   fieldChanged(ev) {
     var el = ev.target;
     this.fieldInput[el.name] = el.value;
   }
 
+  checkboxChanged(ev) {
+    var el = ev.target;
+    this.fieldInput[el.name] = el.checked;
+  }
+
+
+  fieldIsCheckbox(type) {
+    var checkboxTypes = ['checkbox', 'boolean', 'toggle'];
```

Comment:
We should probably reuse the same array at the top for both functions, or maybe refactor this somehow to eliminate the duplicated array.

https://github.com/home-assistant/home-assistant-polymer/blob/fb0b1286d297587e3d80b3ce304ece014338a9be/src/layouts/home-assistant-main.html#L90-L93

Reply:
DRY'd it up in two constants.
This allows for more kinds of fields. (like dates, dropdowns or radio-buttons) 
And it defaults to string types if its none of the other types.
If only polymer would have something like 'dom-else' then we don't need that 'NON_STRING_TYPE' array.